packages/cli/src/util/telemetry/commands/integration/remove.ts

```diff
@@ -1,7 +1,12 @@
 import { TelemetryClient } from '../..';
+import type { TelemetryMethods } from '../../types';
+import type { removeSubcommand } from '../../../../commands/integration/command';
 
-export class IntegrationRemoveTelemetryClient extends TelemetryClient {
-  trackCliArgumentIntegration(v?: string, known?: boolean) {
+export class IntegrationRemoveTelemetryClient
+  extends TelemetryClient
+  implements TelemetryMethods<typeof removeSubcommand>
+{
+  trackCliArgumentIntegration(v: string | undefined, known?: boolean) {
```

Comment:
Why `known?: boolean` instead of `known: boolean | undefined`? The latter _requires_ the argument to be passed, the former doesn't.

Reply:
https://github.com/vercel/vercel/pull/12446/files#r1823797083